app/views/generic_files/_show_actions.html.erb

```diff
@@ -19,7 +19,7 @@
           <%= link_to "Feature", sufia.featured_work_path(@generic_file, format: :json), data: {behavior: 'feature'} %>
         <% elsif @generic_file.featured? %>
           &nbsp;|&nbsp;
-          Featured
+          <%= link_to "Unfeature", sufia.featured_work_path(@generic_file, format: :json), data: {behavior: 'unfeature-page'} %>
```

Comment:
It would be nice to "Unfeature" in the I18n config, but that impacts the JS that's getting appended, so I'm not sure how possible that is.